### PR TITLE
run npm audit fix for @babel/traverse

### PR DESCRIPTION
**Pull request by andrejunges**
Just run `npm audit fix` to remove the vulnerabilities warnings we had related to [@babel/traverse](https://github.com/closeio/best-practices-documentation/security/dependabot/3)

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,26 +1,26 @@
 {
   "name": "@closeio/best-practices-documentation",
   "version": "0.4.2",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```
Comment (bartgryszko) [inline package-lock.json:4]: different npm versions? should we add nvmrc to the repo?

**Review by bartgryszko — APPROVED** [review]
Have you tried running the best-practices-documentation script after the update?

**Comment by andrejunges** [comment]
> Have you tried running the best-practices-documentation script after the update?

My test was installing that specific version in close-ui and running `best-practices:check` and `best-practices:build` - which worked fine as far as I can tell.
```
"@closeio/best-practices-documentation": "closeio/best-practices-documentation#audit-fix",
```

File: package-lock.json (hunk 1 above)
Comment (andrejunges) [reply package-lock.json:4]: Do you think it's worth keeping the same `lockfileVersion`? There's indeed a way to keep that the same with `--lockfile-version`

Comment (andrejunges) [reply package-lock.json:4]: More info about it at https://docs.npmjs.com/cli/v8/using-npm/config#lockfile-version

Comment (treycucco) [reply package-lock.json:4]: I see no issues with updating the lockfile version. But adding an .nvmrc is a great idea!

**Review by treycucco — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (andrejunges) [reply package-lock.json:4]: Agreed. Is it fine to use the same as `close-ui` (`18.16.1`)? or should we go for the current LTS (`20.9.0`)?

Comment (andrejunges) [reply package-lock.json:4]: I'll merge it to address the security warning, and we can update the node version if needed in a separate PR.

**Pull request merged**